apps/onboarding/src/app/result/page.tsx

```diff
@@ -82,34 +84,53 @@ const Volumn = styled.p`
 `;
 
 const Result = () => {
+  const pathname = usePathname();
   const searchParams = useSearchParams();
+  const drinkType = searchParams.get('drinkType');
   const glasses = Number(searchParams.get('glasses'));
   const { name, svg, description, mainColor } = getLevelDetails(glasses);
+  const { isLoading, data } = useQuery(['result'], () => {
+    return axios
+      .get(
+        `https://sulsul.app/api/v1/drinkingLimit/share?drinkType=${drinkType}&glass=${glasses}`
+      )
+      .then((response) => response.data);
+  });
+
+  const shareKakao = () => {
+    window.Kakao.Share.sendScrap({
+      requestUrl: `https://sulsul.app${pathname}?glasses=${glasses}`,
```

Comment:
생각해보니 코드리뷰할 때는 존댓말할게
이렇게는 어떨까요? document.location.href를 사용하면 위에 코드들도 많이 삭제할 수도 있을 거 같아요

![image](https://github.com/mash-up-kr/sulsul-web/assets/61593290/bada5d0d-4c32-4a4a-a4d2-39c09bfe30b8)

![image](https://github.com/mash-up-kr/sulsul-web/assets/61593290/2a267cb0-5e9c-4cb7-9d89-6a04d79450a7)


```suggestion
      requestUrl: document.location.href,
```